diagnostic.md

```diff
@@ -105,18 +157,53 @@ JSON.
 ```
 
 ```sh
-# your answer here
+curl --request PATCH 7 http://www.example.com/donuts/42 \
+--header "Content-Type: application/json" \
+--data '{
+    "name": "Krüller"
+    "done": true
+}'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function(event) {
+  event.preventDefault();
+  $.ajax({
+    url: 'http://www.example.com/donuts/42',
+    type: 'PATCH',
+    data: {
+      "name": "Krüller"
+      'done': true,
```

Comment:
single quotes vs double quotes. This should break but be aware that you are doing it.